core/controllers/suggestion_test.py

```diff
@@ -223,14 +250,33 @@ def test_accept_suggestion(self):
             exploration.states[suggestion_to_accept[
                 'change']['state_name']].content.html,
             suggestion_to_accept['change']['new_value']['html'])
+
+        # Test editor can reject successfully.
```

Comment:
This is in the wrong test suite? (The test name says test accept suggestions).

Reply:
Moved to separate suite.